### ISSUE TITLE
Fix accessibility of feedback target view

### ISSUE THREAD

**Issue by SiniCode** [UniversityOfHelsinkiCS/palaute#1668]
- [x] Add screen reader info that clicking teacher chip opens email
- [x] The x button that removes teacher should be accessible on keyboard
- [x] Active tab should not be indicated only visually
- [x] Tabs might be better navigated with arrow keys instead of tab
- [x] Fix zoom issues on Share, Respondents, Togen, and Logs tabs + organisation logs tab
- [x] Add aria-label that tells the link target for Markdown link in Give counter feedback tab
- [x] Move send button after email selection and add info about optional email to the instructions in Give counter feedback
- [x] In Share tab, add aria-labels to copy buttons that tell what the button copies
- [x] In Answers tab, make all tooltips accessible
- [x] In Continuous feedback tab (teacher): fix contrast of helper text and check it does not overlap with the button on large zoom
- [x] In Continuous feedback tab (student, small course): move consent checkbox above submit button (add info?)
- [x] Add aria-required to required questions
- [x] Fix group selector: tooltip accessibility and focus indicators
- [x] In answers tab, fix download menu's keyboard accessibility

**Comment by SiniCode** [comment]
- The x button that removes teacher should be accessible on keyboard: This is a known accessibility issue in MUI Chip and remains unsolved: https://github.com/mui/material-ui/issues/46753. However, this only affects admin users so will be overlooked for now.
- Fix group selector: Teacher chips on group button tooltips remain inaccessible with keyboard. However, the same teacher chips are keyboard accessible in feedback target information section.
- Add aria-required to required questions: Did not seem to do anything in radio group / group of check boxes. Instead, visually hidden text 'Required' added to question labels that are associated with the required input groups.
